.githooks/setupGitSecrets.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# We cannot modify your .git folder where the hooks are stored in
+# but we can set you default hooksPath so we can apply git-secrets to following hooks:
+# commit-msg
+# pre-commit
+# prepare-commit-msg
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+file="./git-secrets-patterns"
+
+is_patterns_installed() {
+    installedPatterns=$(git config --get-all secrets.patterns | sed ':a;N;$!ba;s/[\n\s]//g')
+    gitSecretsPatterns=$(cat $file | sed ':a;N;$!ba;s/[\n\s]//g')
+    if [[ "$installedPatterns" == "$gitSecretsPatterns" ]]; then
+        return 1 # false
+    else
+        return 0 # true
+    fi
+}
+
+if [[ ! $(command -v git-secrets) ]]; then
+    echo "Warning: awslabs/git-secrets not installed"
```

Comment:
suggestion: add line which contains command to install git hooks

Reply:
there is no general install command but further information with a link to the installation guide could be added